book/src/install.md

```diff
@@ -55,9 +55,16 @@ nix-env -i cargo-dist
 #### cargo-binstall
 
 ```sh
+cargo install cargo-binstall # if you don't have binstall available already
 cargo binstall cargo-dist
 ```
 
+#### cargo-install
+
+```sh
+cargo install cargo-dist
```

Comment:
`cargo install --locked cargo-dist`, as well as mentioning that this will install from source